[Go] feat: update docs for dd-trace-go/v2 (#25897)
DataDog/documentation: content/en/profiler/guide/isolate-outliers-in-monolithic-services.md

})
```

> To specify which label keys you want to use for filtering, add the [WithCustomProfilerLabelKeys][2] (if you are on v2.x of the Go Tracer, refer to the v2 docs [here][3]) option when starting the profiler:

Reviewer: ```suggestion
To specify which label keys you want to use for filtering, add the [WithCustomProfilerLabelKeys][2] ([or WithCustomProfilerLabelKeys v2][3]) option when starting the profiler:
```
